### PR TITLE
Fix NPE when installing a plugin via URL

### LINKED ISSUE #11808: Installing a plugin via URL causes NPE
On `v5.14-RC1` installing a plugin via URL causes an NPE on the server. This effectively prevents admins from using this feature.

### PR DESCRIPTION
**Pull request by hanzei**
#### Ticket Link
Fixes  #11808
Jira ticket: https://mattermost.atlassian.net/browse/MM-17590

### CODE REVIEW

File: api4/plugin.go [hunk 1]
```diff
@@ -127,16 +127,15 @@ func installPluginFromUrl(c *Context, w http.ResponseWriter, r *http.Request) {
...
+	if err != nil {
 		c.Err = model.NewAppError("installPluginFromUrl", "api.plugin.install.reading_stream_failed.app_error", nil, err.Error(), http.StatusBadRequest)
 		return
 	}
 
-	manifest, unpackErr := c.App.InstallPlugin(bytes.NewReader(fileBytes), force)
-
-	if unpackErr != nil {
-		c.Err = unpackErr
+	manifest, appErr := c.App.InstallPlugin(bytes.NewReader(fileBytes), force)
```
Comment (hanzei) [inline api4/plugin.go]: This is not part of the fix. But it's right next to the code I'm changing, hence I'm changing the code to follow the convention of `err` and `appErr`.

**Comment by hanzei** [comment]
Is this fine for `v5.14`?

**Comment by hanzei** [comment]
> Is this fine for `v5.14`?

@marianunez @mickmister Do you know who decides this?

**Comment by marianunez** [comment]
Given that this is a bug fix with minor risk for the release I think the Toolkit team can make the call directly to cherry pick for v.5.14. Thoughts @lieut-data ?

@hanzei It would be helpful to get a corresponding JIRA ticket so we can track it all the way through QA as well within our sprint.

**Comment by lieut-data** [comment]
This should definitely be fixed in v5.14 -- thanks, @marianunez & @hanzei!

**Comment by prapti** [comment]
@hanzei @marianunez Checked with Linda that this PR can be tested post-merge, so once there's a Jira ticket in place, please link it in description above, and we can remove the QA Review label.

**Comment by marianunez** [comment]
Jira Ticket: [MM-17590](https://mattermost.atlassian.net/browse/MM-17590)

**Comment by prapti** [comment]
Thanks @marianunez ! This will be tested post merge.

**Pull request merged**